### DISCUSSION TITLE
Unlock #67 (the issue for non-AVX2 build) ?

### DISCUSSION THREAD

**Discussion by equal-l2** [oven-sh/bun#873] [General]
AVX2 detection in base64 module has been implemented in #824.
I want to know if this has completely solved the build and run on AVX2, so I'd like maintainers to unlock #67 to hear non-AVX2 users.

**Comment by Jarred-Sumner** [comment]
This isn't the main issue facing non-AVX2 support. But I am working on it as we speak. 

Once [this action](https://github.com/oven-sh/bun/actions/runs/2736617399) runs successfully, we should have automated builds for macOS x64, macOS arm64, Linux arm64, Linux x64, Linux x64 for non-avx2 CPUs (`-baseline`) and macOS x64 for non-avx2 CPUs.

**Comment by Jarred-Sumner** [comment]
This is fixed

**Reply by R0N1n-dev** [reply]
Using canary build  as suggested and still getting 'illegal instruction'. cant join discord. Posted github issues in the repo and it keeps getting closed as 'solved , install 0.1.7 or baseline build or canary build' and its still not working. I dont know how to get this across. am starting to just feel real stupid right now. why am i even bothering with this?
